contributing/building-an-amp-extension.md

```diff
@@ -253,9 +253,11 @@ Element styles are loaded when the element script itself is included in
 an AMP doc. You tell AMP which CSS belongs to this element when
 registering the element (see below).
 
-Class names prefixed with `-amp-` are considered private and
+Class names prefixed with `-amp-` and `i-amphtml` are considered private and
```

Comment:
"...are considered private and publishers are not allowed to use to customize (enforced by AMP validator)..."

I think this can be clearer. To "..are considered private and are not allowed for publishers to use for customization." ?

Reply:
We only enforce not allowing `i-amphtml-` as a prefix in the [validator](https://github.com/ampproject/amphtml/blob/master/validator/validator-main.protoascii#L947), not `-amp` as a prefix. We could disallow the `-amp-` prefix as well, please open an issue assigned to me if we want to pursue that.

Reply:
A quick search on the codebase finds that we don't use `-amp` has a prefix anywhere except for all across `<amp-pinterest>`. There are two use cases of `-amp-` in the middle of the class in `amp.css` for `i-amp-scroll-container` and `i-amp-video-mask` referencing https://github.com/ampproject/amphtml/issues/8464, which is a cleanup migration to go from `amp-` to `i-amphtml`. I think it is not necessary to have 2 global namespaces here, and we can just finish the migration to `i-amphtml` by cleaning up these components above.

Reply:
Addressed @alabiaga 's request.

Reply:
Drive-by: If we're enforcing one vs the other, I think migrating to i-amp- makes more sense for file size. These strings are littered all across our codebase, we could save like ~6kb total (a search for i-amphtml in CSS and JS files returns 1533 results, 1533 * 4 = 6,132), but that's not accounting for gzip.